File: even_number_digits.java
public class Solution {
  public int findNumbers(int[] nums){
    int evenNumber = 0;
    for (int num: nums){
      String numStr = Integer.toString(num);
      if (numStr.length() % 2 == 0){
        evenNumber++;
      }
    }
    return evenNumber;
  }
  
}

public static void main(String[] args){
  Solution solution = new Solution();
  int[] nums = {555, 901, 482, 1771};
  System.out.println(solution.findNumbers(nums))
}

File: even_number_digits.py
class Solution:
  def findNumbers(self, nums: list[int]) -> int:
    even_number = 0
    for num in nums:
      num = str(num)

      if len(num) % 2 == 0:
        even_number += 1
    
    return even_number

nums = [555,901,482,1771]
solution = Solution()

print(solution.findNumbers(nums))